torch/onnx/symbolic_opset13.py

```diff
@@ -142,3 +142,26 @@ def reduce_dim(g, self, dim, keepdim, dtype):
     return reduce
 
 sum = _reduce_with_dtype('ReduceSum', 'sum')
+
+@parse_args('v', 'i', 'i', 'i')
+def unsafe_chunk(g, self, chunks, dim, _outputs=None):
+    if _outputs is None:
+        return g.op("SplitToSequence",
+                    self,
+                    g.op("Constant", value_t=torch.tensor(1, dtype=torch.long)),
+                    axis_i=dim, keepdims_i=0)
+
+    size = sym_help._get_tensor_dim_size(self, dim)
```

Comment:
could `size` potentially be a symbol?

Reply:
This part could be implemented more dynamically, but we were looking at the conditional logic in line 160. The dynamic implementation of this part was more complex in this case.